vending-machine/readme.md

```diff
@@ -0,0 +1,114 @@
+## Introduction
+This guide will walk you through the process of building contracts for the vending machine, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
+
+The vending machine allows you to automatically trade an item for another item. For example, if a player gives you one set item depending on the ratio you set it could give the player two of another item.
```

Comment:
Can we give some details into it ? like A vending machine is a SSU in game which can be configured to trade items. and what could the items be

Reply:
Have added this.